plugin-server/src/utils/event.ts

```diff
@@ -19,14 +19,15 @@ export function convertToProcessedPluginEvent(event: IngestionEvent): ProcessedP
     }
 }
 
-export function convertToIngestionEvent(event: ClickHouseEvent): IngestionEvent {
+export function convertToIngestionEvent(event: ClickhouseEventKafka): IngestionEvent {
+    const properties = JSON.parse(event.properties)
```

Comment:
nit:

In `eachMessageIngestion` we parse the nested objects in the function itself, which I think is more clear.

What if for some reason in the future we stop calling `convertToIngestionEvent` and forget we should have parsed the properties?

Tests should catch it etc I guess, so it's more of a style nit

Reply:
Note that typing also catches this as this was the main motivation for ClickhouseEventKafka. That said, let's iterate as needed.